src/mmdata.c

```diff
@@ -20,58 +18,57 @@ mmdata.c
 #include "mmdata.h"
 #include "mminou.h"
 #include "mmpars.h"
-#include "mmcmdl.h" /* Needed for g_logFileName */
-#include "mmpfas.h" /* Needed for g_proveStatement */
-#include "mmwtex.h" /* Needed for SMALL_DECORATION etc. */
+#include "mmcmdl.h" // Needed for g_logFileName
+#include "mmpfas.h" // Needed for g_proveStatement
+#include "mmwtex.h" // Needed for SMALL_DECORATION etc.
 #include "mmfatl.h"
 
 /*E*/long db=0,db0=0,db2=0,db3=0,db4=0,db5=0,db6=0,db7=0,db8=0,db9=0;
-flag g_listMode = 0; /* 0 = metamath, 1 = list utility */
-flag g_toolsMode = 0; /* In metamath: 0 = metamath, 1 = text tools utility */
+flag g_listMode = 0; // 0 = metamath, 1 = list utility
+flag g_toolsMode = 0; // In metamath: 0 = metamath, 1 = text tools utility
 
-/* For use by getMarkupFlag() */
+// For use by getMarkupFlag()
 vstring_def(g_proofDiscouragedMarkup);
 vstring_def(g_usageDiscouragedMarkup);
-flag g_globalDiscouragement = 1; /* SET DISCOURAGEMENT ON */
+flag g_globalDiscouragement = 1; // SET DISCOURAGEMENT ON
 
 vstring_def(g_contributorName);
 
-/* Global variables related to current statement */
+// Global variables related to current statement
 int g_currentScope = 0;
 
 long g_MAX_STATEMENTS = 1;
 long g_MAX_MATHTOKENS = 1;
-long g_MAX_INCLUDECALLS = 2; /* Must be at least 2 (the single-file case) !!!
-                         (A dummy extra top entry is used by parseKeywords().) */
+// Must be at least 2 (the single-file case) !!!
+// (A dummy extra top entry is used by parseKeywords().)
+long g_MAX_INCLUDECALLS = 2; 
 struct statement_struct *g_Statement = NULL;
 long *g_labelKey = NULL;
 struct mathToken_struct *g_MathToken;
 long *g_mathKey = NULL;
 long g_statements = 0, labels = 0, g_mathTokens = 0;
 
 struct includeCall_struct *g_IncludeCall = NULL;
-long g_includeCalls = -1;  /* For eraseSource() in mmcmds.c */
+long g_includeCalls = -1; // For eraseSource() in mmcmds.c
 
 char *g_sourcePtr = NULL;
 long g_sourceLen;
 
-/* Null nmbrString */
+// Null nmbrString
 struct nullNmbrStruct g_NmbrNull = {-1, sizeof(long), sizeof(long), -1};
 
-/* Null pntrString */
+// Null pntrString
 struct nullPntrStruct g_PntrNull = {-1, sizeof(long), sizeof(long), NULL};
 
-temp_nmbrString *nmbrTempAlloc(long size);
```

Comment:
Let's not make any code changes, for my own sanity. You can remove it in another PR

Reply:
Ok, that change was to fix a warning reported in https://github.com/metamath/metamath-exe/issues/158, but I agree it can be done in a separate PR if these ones get merged first.